### PR TITLE
Changing Occupation Position and making copyright year dynamic Fixes …

### PR DESCRIPTION
**Pull request by Harshil-Jani**
## Issue that this pull request solves
 
Closes: #82  

## Proposed changes
This PR consists of changes related to the footer of the website. In footer currently the year is hardcoded as 2020 in copyrights. But, This PR will change the hard coded string to Dynamic Year which will be changed automatically as year will change in future. Refer Issue #82 

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

image

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by praveenscience** [comment]
I'll check this right away tomorrow. Thanks for contributing. As a thumb rule, please start working only after assignment of issue so that your efforts will not go in vain....

**Comment by Harshil-Jani** [comment]
> As a thumb rule, please start working only after assignment of issue so that your efforts will not go in vain....

Opps, apologies. I will take care from now on.

**Comment by praveenscience** [comment]
Running the deployments to see what's happening.

**Comment by Harshil-Jani** [comment]
@praveenscience I have made new commits to the PR which consists of removing the UI changes which were not supposed to be changed as discussed in #81  And also, I have edited the ScreenShot in the PR description. So, Now this PR consists solving only the problem of making the Copyright Year Dynamic.

**Comment by Gurdeep475** [comment]
@Harshil-Jani I am not able to run the project in my local machine due to node-sass incompatibility error, how were able to solve that? Please help me.

**Comment by Harshil-Jani** [comment]
@Gurdeep475  Downgrade your Node Sass to Lower version.
Being more specific go to your clone and run  `npm i node-sass@4.11.0`

File: src/styles/_rezume.scss [hunk 1]
```diff
@@ -37,6 +31,12 @@
     &.summary {
       color: #666;
       text-align: justify;
+      strong.position {
+        font-weight: 550;
```
Comment (praveenscience) [inline src/styles/_rezume.scss]: There's nothing called `550` for this.

**Review by praveenscience — APPROVED** [review]
👍

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -3,12 +3,15 @@ import GitInfo from "react-git-info/macro";
 const gitInfo = GitInfo();
 
 const Footer = () => {
+  const year = new Date().getFullYear();
   return (
     <footer className="page-footer font-small pt-4">
       <div className="footer-copyright text-center py-3">
-        Copyright © 2020 Rezume
+
```
Comment (praveenscience) [inline src/components/Footer/Footer.js:10]: Why do we have unnecessary new lines?

**Review by praveenscience — CHANGES_REQUESTED** [review]

**Review by praveenscience — APPROVED** [review]
I am just wondering, why didn't we do this without the intermediate variable? 👍

**Pull request merged**